api/src/pcapi/scripts/educational/import_educational_institutions_and_deposits.py

```diff
@@ -26,14 +29,18 @@ def import_educational_institutions_and_deposits(
         if not headers or "UAICode" not in headers or "depositAmount" not in headers:
             print("\033[91mERROR: UAICode or depositAmount missing in CSV headers\033[0m")
             return
-        _process_educational_csv(csv_rows, ministry)
+        _process_educational_csv(csv_rows, ministry, educational_year_beginning)
     return
 
 
-def _process_educational_csv(educational_institutions_rows: Iterable[dict], ministry: Ministry) -> None:
-    currentYear = datetime.now().year
+def _process_educational_csv(
+    educational_institutions_rows: Iterable[dict],
+    ministry: Ministry,
+    educational_year_beginning: int = None,
+) -> None:
+    current_year = educational_year_beginning if educational_year_beginning is not None else datetime.now().year
```

Comment:
```suggestion
    current_year = educational_year_beginning or datetime.now().year
```
C'est plus court et aucune date n'est False donc ça aura exactement le même fonctionnement

Reply:
C'est un int ici donc ça pourrait être False si on file 0 😅 
C'est pas un cas trop logique niveau métier mais bon...

Reply:
ha mince en effet du coup oublie mon commentaire (ça m'apprendra a ne pas retenir les types quand je review :sweat_smile: